### PR TITLE
feat: improve logging for better elasticsearch compatibility

### PR DESCRIPTION
**Pull request by lklimek**
## Issue being fixed or feature implemented

For better searching in Elasticsearch, logging keys need to be unified.

## What was done?

1. Unified keys for round and height
2. Registered CommitMessage to fix parsing of WAL logs
3. Moved ping/pong logs to P2PDebug level

## How Has This Been Tested?

With e2e tests + manual investigation

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by shotonoff — COMMENTED** [review]
what do you think if a bit simplify the implementation and avoid declaring interface and modify logger(s).
 it would be easier, if we would use a function or a method like 
``` 
   func StateLogKeyvals(state *State) []interface{} {  …     }  
```
   OR
```
   func (s *State) LogKeyvals() []interface{} {  …  } 
```
And use that 
```
  logger.With(StateLogKeyvals(state)…) 
```
OR
```  
   logger.With(state.LogKeyvals()…)
```
probably, we even don't need to make that functions exportable for some components.
  the benefit of this approach is to avoid declaring interface, implementation of that and adds new behaviour to the logger (tm and nop).
  
Another approach which we can use is to add a method to a struct 
```
 fun (s *State) WithLogParams(with with func(keyvals ...interface{}) Logger) { … }
  ```
   and use
    ```
     state.WithLogParams(logger.With) 
     ```  
  
    Also, I'd prefer using the entire key name rather than the builder function, 
    it will make our life easier if we will need to find a piece of code that adds the parameter.

**Comment by lklimek** [comment]
I've rolled back the changes that increased complexity and just leave some simple improvements here.

File: types/validator_set.go [hunk 1]
```diff
@@ -954,7 +954,7 @@ func (vals *ValidatorSet) VerifyCommit(chainID string, blockID BlockID, stateID
 	blockSignID := commit.CanonicalVoteVerifySignID(chainID, vals.QuorumType, vals.QuorumHash)
 
 	if !vals.ThresholdPublicKey.VerifySignatureDigest(blockSignID, commit.ThresholdBlockSignature) {
-		canonicalVoteBlockSignBytes := commit.StateID.SignBytes(chainID)
+		canonicalVoteBlockSignBytes := commit.CanonicalVoteVerifySignBytes(chainID)
```
Comment (shumkov) [inline types/validator_set.go:957]: It seems this change is not related to logging at all.

Comment (lklimek) [reply types/validator_set.go:957]: It is related, as `canonicalVoteBlockSignBytes` is only used to generate error message in the next line. And this is a bugfix.

File: consensus/reactor.go [hunk 2]
```diff
@@ -1151,7 +1151,14 @@ func (ps *PeerState) SendCommit(commit *types.Commit) bool {
 func (ps *PeerState) PickSendVote(votes types.VoteSetReader) bool {
 	if vote, ok := ps.PickVoteToSend(votes); ok {
 		msg := &VoteMessage{vote}
-		ps.logger.Debug("Sending vote message", "ps", ps, "vote", vote)
+		ps.logger.Debug("Sending vote message",
```
Comment (shotonoff) [inline consensus/reactor.go:1154]: wdyt about using the formatting for multiline argument like this https://github.com/dashevo/tenderdash/blob/a3a43f4a7c8b9703618f0bc88476830a01366775/consensus/state.go#L875-L880
?
the idea is to put the first argument on a separate line without exception

**Review by shotonoff — APPROVED** [review]
LGFM, I have only one concern regarding code-style of calling a function with variadic arguments that allows us write multiline arguments. it would be nice to follow one approach for every case. I like this https://github.com/dashevo/tenderdash/blob/a3a43f4a7c8b9703618f0bc88476830a01366775/consensus/state.go#L875-L880
